frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContext/ticketsTableContext.helpers.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ITemplate, PropertyDefinition, PropertyTypeDefinition } from '@/v5/store/tickets/tickets.types';
+import { compact } from 'lodash';
+import { BaseProperties, IssueProperties } from '@/v5/ui/routes/viewer/tickets/tickets.constants';
+import { TableColumn } from '@controls/resizableTableContext/resizableTableContext';
+
+const TABLE_COLUMNS_INVALID_PROPERTIES = ['view', 'image', 'imageList', 'coords'];
+const TABLE_COLUMNS_DEFAULT_PROPERTIES = ['id', 'modelName', BaseProperties.TITLE];
```

Comment:
Not sure how important this is and if it can be ignored, but now the modelName column is permanent. On staging if you have just one model selected then this column is hdden since it's a bit redundant
![image](https://github.com/user-attachments/assets/b41cc5d8-020e-4667-a658-a6a08b02dda1)